developer_manual/basics/controllers.rst

```diff
@@ -825,6 +825,77 @@ To turn off checks the following *Attributes* can be added before the controller
     * ``@NoTwoFactorRequired``` instead of ``#[NoTwoFactorRequired]``
     * ``@NoCSRFRequired``` instead of ``#[NoCSRFRequired]``
 
+In the following some examples of configurations are given.
+
+Showing an HTML page by the user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A typical app needs an ``index.html`` page to show all content within.
+This page should be visible by all users in the instance.
+Therefore, you need to loosen the restriction from admins only (``#[NoAdminRequired]``).
+Additionally, as the user might not have a CSRF checker cookie set yet, the CSRF checks should be disabled (which is fine as this is a template response).
```

Comment:
This is true for all pages that the user visits directly via the browser, not only template responses.
I'm not sure how to word it, such that it's not understood as "just disable it everywhere when something doesn't work".

Reply:
These are just some (three) examples. We could, however, add a sentence that for the sake of the example, the typical case of a template response is used. At least in NC all HTML pages are somewhat templates AFAIK unless you use lazy rendering or such. Am I wrong here?

Reply:
Right, maybe I was just thinking too much. You can serve custom HTML, but indeed almost everything is template based.